Documentation/FrameworkOverview.md

```diff
@@ -10,198 +10,192 @@ the [Design Guidelines][].
 
 Signals are generally used to represent event streams that are already “in progress”,
-like notifications, user input, etc. As work is performed or data is received, 
-events are _sent_ on the signal, which pushes them out to any observers. 
-All observers see the events at the same time. 
-
-Users must [observe](#observers) a signal in order to access its events. 
-Observing a signal does not trigger any side effects. In other words, 
-signals are entirely producer-driven and push-based, and consumers (observers) 
-cannot have any effect on their lifetime. While observing a signal, the user 
-can only evaluate the events in the same order as they are sent on the signal -
-there is no random access to values of the stream.
-
-Signals can be manipulated by applying [primitives][BasicOperators] to them. 
+like notifications, user input, etc. As work is performed or data is received,
+events are _sent_ on the signal, which pushes them out to any observers.
+All observers see the events at the same time.
+
+Users must [observe](#observers) a signal in order to access its events.
+Observing a signal does not trigger any side effects. In other words,
+signals are entirely producer-driven and push-based, and consumers (observers)
+cannot have any effect on their lifetime. While observing a signal, the user
+can only evaluate the events in the same order as they are sent on the signal—there
+is no random access to values of the stream.
+
+Signals can be manipulated by applying [primitives][BasicOperators] to them.
 Typical primitives to manipulate a single signal like `filter`, `map` and
-`reduce` are available as well as primitives to manipulate multiple signals
-at once (`zip`). Primitives operate only on the `next` events of a signal.
+`reduce` are available, as well as primitives to manipulate multiple signals
+at once (`zip`). Primitives operate only on the `Next` events of a signal.
 The `|>` operator is used to apply primitives to a signal. It can also be used
 to compose basic primitives into more complex ones.
 
-The lifetime of a signal consists of any number of `next` events, followed by
-one terminating event: `error`, `completed` or `interrupted` 
-(but no combination of those).
-Terminating events are not included in the streams values, they must be 
+The lifetime of a signal consists of any number of `Next` events, followed by
+one terminating event, which may be any one of `Error`, `Completed`, or
+`Interrupted` (but not a combination).
+Terminating events are not included in the signal’s values—they must be
 handled specially.
 
 ### Pipes
 
-A **Pipe**, created by `Signal.pipe()`, is a [signal](#signals)
+A **pipe**, created by `Signal.pipe()`, is a [signal](#signals)
 that can be manually controlled.
 
-The method returns a [Signal](#signals) and an [Event Sink](#observers). 
-This signal can be controlled by sending events to the sink. This 
+The method returns a [signal](#signals) and a [observer](#observers).
```

Comment:
All observers are sinks. I think using consistent terminology will be helpful for understanding.